guides_and_processes/engineering_meetings.md

```diff
@@ -0,0 +1,266 @@
+## Catalyst Engineering Team Meetings
+The following is a list of the current recurring development team meetings, their primary purposes, and clear guidelines on what is and is not required for each meeting.
+
+### Meetings
+- [Catalyst Engineering Team Meetings](#catalyst-engineering-team-meetings)
+  - [Meetings](#meetings)
+  - [Summary](#summary)
+  - [Standup (Daily, ~15 min)](#standup-daily-15-min)
+    - [Notes for engineers](#notes-for-engineers)
+  - [Sprint Demo (End of Sprint)](#sprint-demo-end-of-sprint)
+  - [Sprint Retro (End of Sprint)](#sprint-retro-end-of-sprint)
+  - [Sprint Planning (Start of Sprint)](#sprint-planning-start-of-sprint)
+  - [Backlog Grooming](#backlog-grooming)
+  - [Tech Discussion](#tech-discussion)
+  - [Catalyst Team Weekly Meeting](#catalyst-team-weekly-meeting)
+
+
+### Summary
+
+Meeting matrix for roles, note this is just a high level guide, attendance at meetings is determined by the role of each engineer.
+
+| Meeting                       | Architects         | Product Owner        | Eng. Manager         | Backend Developers            | Frontend Developers           | Designers                           | Design Manager                      | Testers/QA                         | SRE             | Operations       |
```

Comment:
Suggest adding a `When` column and moving the bracketed times to it.
Suggest removing the distinction between FE and BE (as we don't segregate the devs this way in practice).  Just have developers.  Any Eng. Meeting devs are required for, both FE and BE devs are required, in the current processes.
It also doesn't well capture the fullstack devs, which currently stradle both Ops and Dev, they should be in this standup as "Devs" to reduce siloing of information.